x-pack/test_serverless/functional/test_suites/observability/cases/attachment_framework.ts

```diff
@@ -20,9 +20,11 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
   const cases = getService('cases');
   const find = getService('find');
 
-  describe('Cases persistable attachments', function () {
-    // security_exception: action [indices:data/write/delete/byquery] is unauthorized for user [elastic] with effective roles [superuser] on restricted indices [.kibana_alerting_cases], this action is granted by the index privileges [delete,write,all]
-    this.tags(['failsOnMKI']);
+  // accessing restricted / system indices directly does not work in serverless
+  // security_exception: action [indices:data/write/delete/byquery] is unauthorized for user
+  // [elastic_serverless] with effective roles [superuser] on restricted indices
+  // [.kibana_alerting_cases]
```

Comment:
My PR here https://github.com/elastic/kibana/pull/168924 is going to fix accessing the system index directly. We will use the saved objects API to remove our data. Could you please wait for my PR to get merged and not skip the Cases tests?